config/routes.rb

```diff
@@ -43,6 +43,22 @@
     end
   end
 
+  scope '/request_statuses' do
```

Comment:
Instead of `/request_statuses` and `/medical_statuses`, these could be `/statuses` and `/statuses/medical`.